expression/builtin_time_vec.go

```diff
@@ -558,11 +558,30 @@ func (b *builtinSubDateAndStringSig) vecEvalString(input *chunk.Chunk, result *c
 }
 
 func (b *builtinMinuteSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinMinuteSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETDuration, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err = b.args[0].VecEvalDuration(b.ctx, input, buf); err != nil {
+		return err
```

Comment:
This is not equivalent to the original implementation.
https://github.com/pingcap/tidb/blob/master/expression/builtin_time.go#L904

Reply:
To match the original implementation, should I use `EvalDuration` on each row? I don't know how to get the error associated to a row with `VecEvalDuration`.

Reply:
When I was exploring through `vecEvalDuration` implementations, I found `builtinCastIntAsDurationSig` returns error immediately when errors except overflow error encountered, ignoring the rest of rows, instead of appending warnings and setting result to null (https://github.com/pingcap/tidb/blob/master/expression/builtin_cast_vec.go#L51). If errors are handled correctly there, above codes shouldn't be a problem.

Reply:
Here is not fully compatible with the original implementation. After discussing with @qw4990 , we decided to adopt your implementation first.